schedule/migration/s390x_regression_test_online.yaml

```diff
@@ -0,0 +1,84 @@
+name:          s390x_regression_test_online.yaml
+description:    >
```

Comment:
Better add notes for which specific setting needed if you need use this yaml file.
Also setting name can be add extra prefix such as REGRESSON_xxx

Reply:
Updated.